app/main.py

```
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    command_list = command.split()
+
+    if (len(command_list) != 3
+            or command_list[0] != "cp"
+            or command_list[1] == command_list[2]):
```

The condition `command_list[1] == command_list[2]` checks if the source and destination files are the same. While this prevents copying a file onto itself, it might be more informative to raise an error or log a message instead of silently returning, to inform the user why the operation was not performed.